docs/vision.md

```diff
@@ -72,12 +72,65 @@ moved to GraphQL and (the yet young) Apollo client to embrace their rich and
 innovative ecosystem.
 
 Since then, we worked hard to build a product that supports a wide range of
-eCommerce features
+eCommerce features. We kept our focus on Magento2 which is the platform we
+knew most and which allows to support a lot of eCommerce use cases.
 
-> **TODO** To be continued
+**Front-Commerce is in production as a frontend for Magento2 stores since
+February 2018. It supports most of the Magento Opensource edition.**
+It is composed of a React application, a GraphQL middleware using Magento2
+Web APIs, Redis for caching and ElasticSearch for search — more details on
+[Front-Commerce’s architecture documentation page](architecture-overview.md).
 
-## What do we want it to be
+We wanted to be our first integrators (dogfooding), and to create a few stores with
+Front-Commerce before distributing the product to the world.
+
+Being in production means that we focused on features, themes, payments,
+deployments, caching, logging… and everything you would expect from such a
+solution. **That makes us the most mature « eCommerce PWA theme » in our field.**
+
+We are now working on technical documentation and refactoring (v1.0), along with
+business and legal things to start selling license to customers and help
```

Comment:
There's a lot happening here. Maybe we should try to split it in two parts ?
1. Distribution ready
2. Legal/License/Partners ready

Reply:
also there is a typo : license**s** (if the phrasing stays the same)